content/tutorials/use-simple-xrp-payments/send-xrp.md

```diff
@@ -0,0 +1,593 @@
+
+<div class="interactive-block" id="interactive-check">
+<div class="interactive-block-inner">
+  <div class="breadcrumbs-wrap">
+    <ul class="breadcrumb prereqs">
+      <li class="breadcrumb-item active bc-generate"><a href="#interactive-generate">Generate</a></li>
+      <li class="breadcrumb-item disabled bc-connect"><a href="#interactive-connect">Connect</a></li>
+      <li class="breadcrumb-item disabled bc-prepare"><a href="#interactive-prepare">Prepare</a></li>
+      <li class="breadcrumb-item disabled bc-sign"><a href="#interactive-sign">Sign</a></li>
+      <li class="breadcrumb-item disabled bc-submit"><a href="#interactive-submit">Submit</a></li>
+      <li class="breadcrumb-item disabled bc-wait"><a href="#interactive-wait">Wait</a></li>
+      <li class="breadcrumb-item disabled current bc-check"><a href="#interactive-check">Check</a></li>
+    </ul><!--/.breadcrumb.prereqs-->
+  </div><!--/.breadcrumbs-wrap-->
+  <button id="get-tx-button" class="btn btn-primary connection-required"
+    title="Connection to Test Net required" disabled>Check transaction status</button>
+  <div id="get-tx-output"></div>
+</div>
+</div>
+<script type="application/javascript">
+  $("#get-tx-button").click( async function() {
+    // Wipe previous output
+    $("#get-tx-output").html("")
+
+    const txID = $("#signed-tx-hash").text()
+    const earliestLedgerVersion = parseInt($("#earliest-ledger-version").text(), 10)
+
+    const tx = await api.getTransaction(txID, {minLedgerVersion: earliestLedgerVersion})
+
+    $("#get-tx-output").html(
+      "<div><strong>Transaction result:</strong> " +
+      tx.outcome.result + "</div>" +
+      "<div><strong>Balance changes:</strong> <pre><code>" +
+      JSON.stringify(tx.outcome.balanceChanges, null, 2) +
+      "</pre></code></div>"
+    )
+  })
+</script>
+
+## Differences for Production
+
+To send an XRP payment on the production XRP Ledger, the steps you take are largely the same. However, there are some key differences in the necessary setup:
+
+- [Getting real XRP isn't free.](#getting-a-real-xrp-account)
+- [You must connect to a server that's synced with the production XRP Ledger network.](#connecting-to-the-production-xrp-ledger)
+
+### Getting a Real XRP Account
+
+This tutorial uses a button to get an address that's already funded with Test Net XRP, which only works because Test Net XRP is not worth anything. For actual XRP, you need to get XRP from someone who already has some. (For example, you might buy it on an exchange.) You can generate an address and secret that'll work on either production or the test net using RippleAPI's [generateAddress() method](rippleapi-reference.html#generateaddress):
+
+```js
+const generated = api.generateAddress()
+console.log(generated.address) // Example: rGCkuB7PBr5tNy68tPEABEtcdno4hE6Y7f
+console.log(generated.secret) // Example: sp6JS7f14BuwFY8Mw6bTtLKWauoUs
```

Comment:
I wanted at least one example of an un-masked secret so that users could see what one _actually_ looks like. This is the same secret that's used in the RippleAPI reference docs for `generateAddress()`.